src/videojs.ads.js

```diff
@@ -43,13 +43,35 @@ var
   },
 
   /**
+   * Returns a boolean indicating if given player is in live mode.
+   */
+  isLive = function(player) {
+    if (player.duration() === Infinity) {
+      return true;
+    } else if (videojs.browser.IOS_VERSION === "8" && player.duration() === 0) {
```

Comment:
@gkatsev Should video.js be handling this discrepancy? Seems like the kind of problem we try to abstract over.